domain/src/Commands/DomainCommands.php

```diff
@@ -0,0 +1,1288 @@
+      $changed = FALSE;
+      // Multivalue fields are used, so check each one.
+      foreach ($entity->get($field) as $k => $item) {
+        // NB: NOT strict ==
+        if ($item->target_id == $old_domain->id()) {
+
+          if ($this->is_dry_run) {
+            $this->logger()->info(dt('DRYRUN: Update domain membership for entity @id to @new.',
+              [ '@id' => $entity->id(), '@new' => $new_domain->id() ]));
+
+            // Don't set changed, so don't save either.
+            continue;
+          }
+
+          $changed = TRUE;
+          $item->target_id = $new_domain->id();
+        }
+      }
+      if ($changed) {
+        $entity->save();
+      }
+    }
+    return count($entities);
+  }
+
+  /**
+   * Return the Domain object corresponding to a policy string.
+   *
+   * @param string $policy
+   *   In general one of 'prompt' | 'default' | 'ignore' or a domain entity
+   *   machine name, but this function does not process 'prompt'.
+   *
+   * @return \Drupal\Core\Entity\EntityInterface|\Drupal\domain\DomainInterface|null
+   * @throws \Drupal\domain\Commands\DomainCommandException
+   */
+  protected function getDomainInstanceFromPolicy(string $policy) {
+    $domain_storage = $this->getStorage();
+    switch($policy) {
+      case 'default':
+        $new_domain = $domain_storage->loadDefaultDomain();
+        break;
+
+      case 'prompt':
+      case 'ignore':
+        return NULL;
+
+      default:
+        $new_domain = $domain_storage->load($policy);
+        break;
+    }
+    return $new_domain;
+  }
+
+  /**
+   * Reassign entities of the supplied type to the $policy domain.
+   *
+   *  $options = [
+   *    'entity_filter' => 'node',
+   *    'policy' => 'prompt' | 'default' | 'ignore'
+   *    'multidomain' => FALSE,
+   *  ];
+   *
+   * @param DomainInterface[] $domains
+   *   List of the domains to reassign content away from.
+   *
+   * @throws \Drupal\domain\Commands\DomainCommandException
+   */
+  protected function reassignLinkedEntities($domains, array $options = ['chatty' => null, 'policy' => null]) {
+    $entity_typenames = $this->findDomainEnabledEntities();
+    // DOMAIN_ADMIN_FIELD too??
+    $field_names = [DOMAIN_ACCESS_FIELD, DOMAIN_SOURCE_FIELD];
+
+    $new_domain = $this->getDomainInstanceFromPolicy($options['policy']);
+    if (empty($new_domain)) {
+      throw new DomainCommandException('invalid destination domain');
+    }
+
+    // For each entity type...
+    $exceptions = FALSE;
+    foreach ($entity_typenames as $name) {
+      if (empty($options['entity_filter']) || $options['entity_filter'] === $name) {
+
+        // For each domain being reassigned from...
+        foreach ($domains as $domain) {
+
+          // And, For each domain field ...
+          foreach ($field_names as $field) {
+            $ids = $this->enumerateDomainEntities($name, $domain->id(), $field);
+            if (!empty($ids)) {
+
+              try {
+                if ($options['chatty']) {
```

Comment:
Isn't `verbose` the most standard option name?

Reply:
Sadly drush itself uses verbose, so although it is available to be used by commands you end up seeting lots of drush internal verbose stuff if you use it.